<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BioPrintClass</class>
 <widget class="QMainWindow" name="BioPrintClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BioPrint</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,3,25,1" columnstretch="1,3,0">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item row="1" column="0" colspan="3">
     <layout class="QHBoxLayout" name="TopLayout">
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="LeftLayout" stretch="3,2">
      <item>
       <layout class="QVBoxLayout" name="LeftLayout_U"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="LeftLayout_D" stretch="3,2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="LeftLayout_D_U">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="LeftLayout_D_D"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="1" colspan="2">
     <layout class="QHBoxLayout" name="RightLayout" stretch="2,1">
      <item>
       <layout class="QVBoxLayout" name="RightLayout_L" stretch="2,1">
        <item>
         <layout class="QHBoxLayout" name="RightLayout_L_U"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="RightLayout_L_D">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="RightLayout_R">
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="3">
     <layout class="QHBoxLayout" name="BottomLayout"/>
    </item>
    <item row="0" column="0" colspan="3">
     <layout class="QHBoxLayout" name="TitleLayout"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BioPrint.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>setModelWidget()</slot>
  <slot>setProjectWidget()</slot>
  <slot>setModelRect()</slot>
  <slot>myslot()</slot>
 </slots>
</ui>
